### ISSUE TITLE
Expand the cross validation scripts to remove multiple images from each class

### ISSUE THREAD

**Issue by bentsherman** [CUFCTFACE/face-recognition#8]
Currently our cross-validation scripts create a test set by removing one image from each class in the training set. However, it would be useful to have these scripts remove 2 or 3 or more images from each class in order to make a "harder" test set. A parameter should be added to each of these scripts to specify this value.
